cmd/kubectl-gadget/top/top.go

```diff
@@ -15,18 +15,31 @@
 package top
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
+	"strconv"
 	"strings"
-
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
+	"sync"
+	"time"
 
 	"github.com/spf13/cobra"
+	"golang.org/x/term"
+
+	commonutils "github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/inspektor-gadget/inspektor-gadget/pkg/apis/gadget/v1alpha1"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/top"
 )
 
 type CommonTopFlags struct {
+	commonFlags utils.CommonFlags
```

Comment:
The `TraceConfig` wants a pointer to a  `utils.CommonFlags` in line `125` of file `top/top.go`. Therefore i have to declare it as a member, right?
Or is there any other way?

Reply:
You should still be able to get a pointer using @commonTopFlags.CommonFlags.

Reply:
I meant that you could [embed it](https://go.dev/doc/effective_go#embedding):

```golang
type CommonTopFlags struct {
	*utils.CommonFlags
	...
}
```

In this way, instead of `g.commonTopFlags.commonFlags.Timeout`, you could use `g.commonTopFlags.Timeout`, which looks better, IMHO 🙂.

Reply:
And yes, as Mick said, you can still get it.

Reply:
Seems i tried with a lowercase `commonFlags` instead of a uppercase `CommonFlags`. Indeed you can still get it 😅. 
I also like the embedded access way more 👍